src/test/java/com/epam/tc/hw2/exercise2/SiteSelectorsTests.java

```diff
@@ -0,0 +1,288 @@
+                      .as("Correct log message when selected Selen selector")
+                      .contains("metal: value changed to Selen");
+
+        // test dropdown
+
+        // change current selection to prevent intersection with first check
+        dropdownSelectorColors.selectByVisibleText("Yellow");
+
+        dropdownSelectorColors.selectByVisibleText("Red");
+        selectedColor = dropdownSelectorColors.getFirstSelectedOption();
+        softAssertions.assertThat(selectedColor.getText())
+                      .as("Selected color is Red")
+                      .isEqualTo("Red");
+        softAssertions.assertThat(
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Red in dropdown menu 'colors'")
+                      .contains("Colors: value changed to Red");
+
+        dropdownSelectorColors.selectByVisibleText("Green");
+        selectedColor = dropdownSelectorColors.getFirstSelectedOption();
+        softAssertions.assertThat(selectedColor.getText())
+                      .as("Selected color is Green")
+                      .isEqualTo("Green");
+        softAssertions.assertThat(
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Green in dropdown menu 'colors'")
+                      .contains("Colors: value changed to Green");
+
+        dropdownSelectorColors.selectByVisibleText("Blue");
+        selectedColor = dropdownSelectorColors.getFirstSelectedOption();
+        softAssertions.assertThat(selectedColor.getText())
+                      .as("Selected color is Blue")
+                      .isEqualTo("Blue");
+        softAssertions.assertThat(
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Blue in dropdown menu 'colors'")
+                      .contains("Colors: value changed to Blue");
+
+        dropdownSelectorColors.selectByVisibleText("Yellow");
+        selectedColor = dropdownSelectorColors.getFirstSelectedOption();
+        softAssertions.assertThat(selectedColor.getText())
+                      .as("Selected color is Yellow")
+                      .isEqualTo("Yellow");
+        softAssertions.assertThat(
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Yellow in dropdown menu 'colors'")
+                      .contains("Colors: value changed to Yellow");
+
+        // 10. Close Browser
+        webDriver.close();
+        assertThatThrownBy(() -> webDriver.getTitle()).as("Browser is closed")
+                                                      .isInstanceOf(NoSuchSessionException.class);
+
+        softAssertions.assertAll();
```

Comment:
Why do you check all of the elements? What were required?

Reply:
From requirements
Step
Assert that for each checkbox there is an individual log row and value is corresponded to the status of checkbox
Expected result
Log rows are displayed and isplayed and checkbox name and its status are corresponding to selected

So I thought that I should check "each checkbox". 
Or you mean that I can gather log messages into List of strings and compare it with expected List of strings (according to clicking order, starts from step 6) ?
Looks like I totally didn't catch the task :( please eplain

Reply:
Fixed